awscli/examples/global_options.rst

```diff
@@ -0,0 +1,34 @@
+``--debug`` (boolean)
+Turn on debug logging.
```

Comment:
A bit of a nitpick but it would be nice if there were two newlines between the description and the next global option:
```
``--debug`` (boolean)
Turn on debug logging.

``--endpoint-url`` (string)
Override command's default URL with the given URL.
```
Mainly with all of the newlines reduced to one, it is a little difficult to read because everything is combined together.